### PR TITLE
chore(locales): support for quests

### PR DESCRIPTION
**Pull request by Moore2021**

### CODE REVIEW

**Comment by Moore2021** [comment]
If this is how we want todo the quest locales there are two things that need to be done before shipping to production.

1) Delete the "quests" directory, should only have a quest file named en.json in it
2) Alter the table on the database, SQL statement is in the whiteboard channel on discord.

File: src/commands/user/quests.js [hunk 1]
```diff
@@ -23,9 +23,9 @@ module.exports = {
 		
 		const questSession = new Quest(client, reply)
 		await questSession.start(sessionID, user, locale, messageRef)
-		if (questSession.getSessionActive) return
```
Comment (klerikdust) [inline src/commands/user/quests.js:26]: Is this temporary for testing?

File: src/libs/localizer.js [hunk 2]
```diff
@@ -30,4 +30,9 @@ const getTargetLocales = (targetLang, fallbackLang = `en`) => {
   return locales[TLang]
 }
 
-module.exports = { getTargetLocales }
+const availableLocales = {
```
Comment (klerikdust) [inline src/libs/localizer.js:33]: should we plan to use a master table in db? for convenience .. and possibly better control over the variables of each of the languages.

File: src/commands/user/quests.js (hunk 1 above)
Comment (Moore2021) [reply src/commands/user/quests.js:26]: Yes, lines 26 and 27 should be uncommented, they were commented for testing reasons

File: src/libs/localizer.js (hunk 2 above)
Comment (Moore2021) [reply src/libs/localizer.js:33]: So reason for this is, we don't need a db call for this as it won't ever change unless we add or remove a language

**Review by klerikdust — COMMENTED** [review]
👍✨

File: src/libs/localizer.js (hunk 2 above)
Comment (klerikdust) [reply src/libs/localizer.js:33]: understandable

**Comment by klerikdust** [comment]
@Moore2021 just confirming, this is not final right?

**Comment by Moore2021** [comment]
> @Moore2021 just confirming, this is not final right?

Not final as there is prereq for production before this will work without errors

**Pull request merged**